bindings/java/java_code/build.gradle

```diff
@@ -7,8 +7,8 @@ plugins {
     id 'org.jreleaser' version '1.12.0'
 }
 
-group = 'io.github.crate-crypto.peerdas-kzg'
-version = '0.0.1' // x-release-please-version
+group = 'io.github.crate-crypto'
+version = '0.0.7' // x-release-please-version
```

Comment:
This is okay to be out of sync with the current repo, it was being used so that I could push test versions to maven